GET request results in memory error on local Rekor server

When I am querying a local Rekor server, I'm able to search for an artifact I uploaded, but using the resulting UUID in a GET request results in a memory error. This does not happen when querying the public Rekor server.

```
[rekor@fedora rekor]$ rekor-cli search --artifact tests/test_file.txt --rekor_server http://127.0.0.1:3000
Found matching entries (listed by UUID):
d2f305428d7c222d7b77f56453dd4b6e6851752ecacc78e5992779c8f9b61dd9
```

```
[rekor@fedora rekor]$ rekor-cli get --uuid d2f305428d7c222d7b77f56453dd4b6e6851752ecacc78e5992779c8f9b61dd9 --rekor_server http://127.0.0.1:3000
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xb0b280]

goroutine 1 [running]:
github.com/sigstore/rekor/cmd/rekor-cli/app.parseEntry(0xc000040280, 0x40, 0x0, 0xb7fe20, 0xc00004a380, 0xc0000364d8, 0xc00004a390, 0xc0000364f0, 0xc0000780a0, 0xc0000780a0, ...)
	/home/rekor/go/src/github.com/sigstore/rekor/cmd/rekor-cli/app/get.go:153 +0x1c0
github.com/sigstore/rekor/cmd/rekor-cli/app.glob..func2(0xc0003c9b00, 0x0, 0x4, 0xc00021fd58, 0x873365, 0xc00011f600, 0xc00021fd30)
	/home/rekor/go/src/github.com/sigstore/rekor/cmd/rekor-cli/app/get.go:129 +0x47d
github.com/sigstore/rekor/cmd/rekor-cli/app/format.WrapCmd.func1(0x118e900, 0xc0003c9b00, 0x0, 0x4)
	/home/rekor/go/src/github.com/sigstore/rekor/cmd/rekor-cli/app/format/wrap.go:33 +0x5e
github.com/spf13/cobra.(*Command).execute(0x118e900, 0xc0003c9ac0, 0x4, 0x4, 0x118e900, 0xc0003c9ac0)
	/home/rekor/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:860 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x118f080, 0xc00005a740, 0x46e485, 0xc000000180)
	/home/rekor/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/rekor/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/sigstore/rekor/cmd/rekor-cli/app.Execute()
	/home/rekor/go/src/github.com/sigstore/rekor/cmd/rekor-cli/app/root.go:52 +0x31
main.main()
	/home/rekor/go/src/github.com/sigstore/rekor/cmd/rekor-cli/main.go:21 +0x25
```

--- a/cmd/rekor-cli/app/get.go
+++ b/cmd/rekor-cli/app/get.go
@@ -150,13 +150,16 @@ func parseEntry(uuid string, e models.LogEntryAnon) (interface{}, error) {
 	}
 
 	obj := getCmdOutput{
-		Attestation:     string(e.Attestation.Data),
-		AttestationType: e.Attestation.MediaType,
-		Body:            eimpl,
-		UUID:            uuid,
-		IntegratedTime:  *e.IntegratedTime,
-		LogIndex:        int(*e.LogIndex),
-		LogID:           *e.LogID,
+		Body:           eimpl,
+		UUID:           uuid,
+		IntegratedTime: *e.IntegratedTime,
+		LogIndex:       int(*e.LogIndex),
+		LogID:          *e.LogID,
+	}
+
+	if e.Attestation != nil {
+		obj.Attestation = string(e.Attestation.Data)
+		obj.AttestationType = e.Attestation.MediaType
 	}
 
 	return &obj, nil

